failed to download remote server release: 500

### Describe the bug / provide steps to reproduce it

Failed to download remote server binary (version: 0.166.2, os: linux, arch: x86_64): failed to download remote server release: 500

### Zed Version and System Specs

0.166.2 pacman x86_64 archlinux null

### Screenshot

![Image](https://github.com/user-attachments/assets/c116397c-735a-4603-9534-33607769ea74)

### If applicable, attach your Zed.log file to this issue.

```
2024-12-22T14:33:38.989081679+08:00 [ERROR] Failed to download binary on server, attempting to upload server: failed to run command: wget: unrecognized option: max-redirect=5
BusyBox v1.37.0 (2024-12-13 21:18:49 UTC) multi-call binary.

Usage: wget [-cqS] [--spider] [-O FILE] [-o LOGFILE] [--header STR]
	[--post-data STR | --post-file FILE] [-Y on/off]
	[-P DIR] [-U AGENT] [-T SEC] URL...

Retrieve files via HTTP or FTP

	--spider	Only check URL existence: $? is 0 if exists
	--header STR	Add STR (of form 'header: value') to headers
	--post-data STR	Send STR using POST method
	--post-file FILE	Send FILE using POST method
	-c		Continue retrieval of aborted transfer
	-q		Quiet
	-P DIR		Save to DIR (default .)
	-S    		Show server response
	-T SEC		Network read timeout is SEC seconds
	-O FILE		Save to FILE ('-' for stdout)
	-o LOGFILE	Log messages to FILE
	-U STR		Use STR for User-Agent header
	-Y on/off	Use proxy

2024-12-22T14:33:38.990207985+08:00 [INFO] downloading zed-remote-server linux x86_64 version 0.166.2
2024-12-22T14:33:38.991010467+08:00 [ERROR] Failed to open project: Failed to download remote server binary (version: 0.166.2, os: linux, arch: x86_64): error sending request for url (https://zed.dev/api/releases/stable/0.166.2/zed-remote-server-linux-x86_64.gz?update=1)
```

--- a/crates/remote/src/ssh_session.rs
+++ b/crates/remote/src/ssh_session.rs
@@ -1910,7 +1910,6 @@ impl SshRemoteConnection {
                     .run_command(
                         "wget",
                         &[
-                            "--max-redirect=5",
                             "--method=GET",
                             "--header=Content-Type: application/json",
                             "--body-data",

